sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemResponse.java

```diff
@@ -26,9 +26,8 @@
      *
      * @return the resource
      */
-    @SuppressWarnings("unchecked")
     public T getResource() {
```

Comment:
@kushagraThapar  I thought we changed this to getItem(). No?

Reply:
Yes, I thought so too, just checked, we only changed it in`CosmosAsyncItemResponse` and not this one. We should change it too.. 
@anuchandy  -> Can you please rename this API to `getItem()` - the same way we have it in `CosmosAsyncItemResponse`

Reply:
This is changed to getItem() in Async but not in Sync, and should be changed

Reply:
@kushagraThapar sure, renamed to `getItem`